.github/workflows/bindings-python.yml

```diff
@@ -40,7 +40,7 @@ concurrency:
 
 jobs:
   lint:
-    name: PEP8 style check
+    name: Python PEP8 format and lint check
```

Comment:
this one says lint, but we dont lint here (also on line 61)

Reply:
yeah, lefover, fixed